.cirrus.yml

```
@@ -196,3 +196,24 @@ task:
     - rm /etc/ld.so.cache
     - ./ci/cirrus.sh
   << : *CAT_LOGS
+
+task:
+  name: "x86_64 (mingw32-w64): Windows (Debian stable, Wine)"
+  container:
+    dockerfile: ci/linux-debian.Dockerfile
+    cpu: 1
+    memory: 1G
+  env:
+    WINE_CMD: wine
```

I think this would hide the warning.
```suggestion
    WINE_CMD: wine64-stable
```

---

IIRC I tried that, but it didn't work. I think invoking wine64 directly doesn't start wineserver.